packages/client/ansible-collection-theforeman-foreman/ansible-collection-theforeman-foreman.spec

```
@@ -0,0 +1,56 @@
+%global debug_package %{nil}
+
+%global collection_namespace theforeman
+%global collection_name foreman
+%global collection_folder %{_datadir}/ansible/collections/ansible_collections/%{collection_namespace}/%{collection_name}
```

Folder is a bit of a Windows naming convention.
```suggestion
%global collection_directory %{_datadir}/ansible/collections/ansible_collections/%{collection_namespace}/%{collection_name}
```